src/ApplicationInstanceCommands.cpp

```diff
@@ -263,8 +263,25 @@ std::multimap<std::string,ServiceInterface> getServices(const SharedFileHandle&
 				for(const auto& path : rule[protocol]["paths"].GetArray()){
 					if(!path.HasMember("backend"))
 						continue;
-					std::string serviceName=path["backend"]["serviceName"].GetString();
-					int servicePort=path["backend"]["servicePort"].GetInt();
+
+					std::string serviceName;
+					int servicePort;	
```

Comment:
`servicePort` should probably be initialized, in case some future version of Kubernetes changes this structure again, causing neither of the following conditions to match. `serviceName ` being default initialized to the empty string probably prevents the undefined, unintialized value of `servicePort` from being used, but it would be good to be certain that it cannot cause trouble.

Reply:
And what about servicePort, should I set that to zero?